mmcv/runner/builder.py

```diff
@@ -1,7 +1,23 @@
+import copy
+
 from ..utils import Registry, build_from_cfg
 
 RUNNERS = Registry('runner')
+RUNNER_BUILDERS = Registry('runner builder')
+
+
+def build_runner_constructor(cfg):
+    return build_from_cfg(cfg, RUNNER_BUILDERS)
```

Comment:
We can simply use `RUNNER_BUILDERS.build(cfg)`

Reply:
Got it